### DISCUSSION TITLE
where can I find the Crazyfile's Mechanical drawings？

### DISCUSSION THREAD

**Discussion by cvirxsc** [bitcraze/discussions#496] [Q&A] [answered]
where can I find the Crazyfile's Mechanical drawings？I want to use it in gazebo environment.

**Comment by knmcguire** [comment, marked as answer]
If you want to use it in Gazebo, you could use the meshes in this repo here: https://github.com/bitcraze/crazyflie-simulation

There should be also a model.sdf there in the gazebo ignition folder

**Reply by cvirxsc** [reply]
Oh yes, yes, this is what I am looking for, thank you very much for your help
